zerver/views/webhooks/trello/card_actions.py

```diff
@@ -0,0 +1,197 @@
+from typing import Dict, Tuple, Any
+from datetime import datetime
+from .exceptions import UnknownUpdateCardAction
+from .templates import TRELLO_SUBJECT_TEMPLATE, TRELLO_MESSAGE_TEMPLATE
+
+SUPPORTED_CARD_ACTIONS = [
+    'updateCard',
+    'createCard',
+    'addLabelToCard',
+    'removeLabelFromCard',
+    'addMemberToCard',
+    'removeMemberFromCard',
+    'addAttachmentToCard',
+    'addChecklistToCard',
+    'commentCard'
+]
```

Comment:
I think card actions should be typed `text_type`, just like board actions. So prefix each of these strings with a `u`.